ql/src/test/results/clientpositive/llap/autoColumnStats_10.q.out

```diff
@@ -205,7 +205,7 @@ Retention:          	0
 #### A masked pattern was here ####
 Table Type:         	MANAGED_TABLE       	 
 Table Parameters:	 	 
-	COLUMN_STATS_ACCURATE	{\"BASIC_STATS\":\"true\",\"COLUMN_STATS\":{\"insert_num\":\"true\"}}
+	COLUMN_STATS_ACCURATE	{\"BASIC_STATS\":\"true\",\"COLUMN_STATS\":{\"c1\":\"true\",\"c2\":\"true\",\"insert_num\":\"true\"}}
```

Comment:
In this test an alter table statement changes the type of columns `c1` and `c2` and it removes the column stats entries from HMS backend.
https://github.com/apache/hive/blob/e6c6029439f831545fe5fa267452b662c3a71403/standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/HiveAlterHandler.java#L1085-L1089

from `tinyint` to `string`
```
alter table p_n1 replace columns (insert_num int, c1 STRING, c2 STRING);
```

The insert follows the alter 
```
insert into p_n1 values (1,22,333);
```
can not merge the stats for `c1` and `c2` columns because those are not exists.
So the 
```
\"c1\":\"true\",\"c2\":\"true\"
```
is not true.

Reply:
I dont think we require to update the native tables in anyway since table level updation takes care of all cases.
I have made the update specific to non-native tables only. Done.